salt/modules/chocolatey.py

```diff
@@ -930,8 +930,14 @@ def version(name, check_remote=False, source=None, pre_versions=False):
         available = list_(narrow=name, pre_versions=pre_versions, source=source)
 
         for pkg in packages:
-            packages[pkg] = {'installed': installed[pkg],
-                             'available': available[pkg]}
+            # Grab the current version from the package that was installed
+            packages[pkg] = {'installed': installed[pkg]}
```

Comment:
I wonder - would this be better as `'available': available.get(pkg)`?

It produces an output like this:
```
local:
    ----------
    hosts:
        ----------
        available:
            None
        installed:
            - 1.1.0
```

Which feels better than:

```
local:
    ----------
    hosts:
        ----------
        installed:
            - 1.1.0
```